src/app/interceptors/jwt/jwt.interceptor.spec.ts

```diff
@@ -1,6 +1,6 @@
 import { TestBed } from '@angular/core/testing';
 import { HttpClientTestingModule } from '@angular/common/http/testing';
-import { httpHandlerSpy, httpRequestSpy } from './spies';
+import { httpHandlerSpy, httpRequestSpy } from '../../helpers/tests/spies';
```

Comment:
@kdaglas we can transform this into an absolute path

Reply:
on it... thanks